assets/src/domain/wpPluginsPage/index.tsx

```diff
@@ -25,8 +23,7 @@ const handleDeactivationClick = (e: JQuery.ClickEvent<HTMLElement>, el: any): vo
 /**
  * EE caffeinated click capture but only if feature is active
  */
-const ok = true;
```

Comment:
weird variable

Reply:
was still there, I added it for your test of Exit Survey